lib/feature/wallet/token_wallet_send/bloc/token_wallet_send_bloc.dart

```diff
@@ -65,6 +65,7 @@ class TokenWalletSendBloc
   late Currency tokenCurrency;
 
   late UnsignedMessage unsignedMessage;
+  UnsignedMessage? _unsignedMessage;
```

Comment:
Why do you store non-final thing in the bloc itself instead of it's state? This applies to several more fields of this and TonConfirmTransactionBloc classes.